### PR TITLE
Zd 171

### PR DESCRIPTION
**Pull request by bazzika**

### CODE REVIEW

**Comment by bazzika** [comment]
list examples

File: docs/spec.md [hunk 1]
```diff
@@ -35,8 +35,9 @@ To narrow down a scope we can disable examples which not supported yet
  * @todo #136:2d/UX Open Spec - list of screens in Zeplin
  *
  */
- https://zpl.io/aweA0Y1
- zpl://screen?pid=5b424f034d50f36965a0be52&sid=5e270b38baa2047ef1e9112b
+https://zpl.io/2yeQJ4q
```
Comment (nesterone) [inline docs/spec.md]: plz, create a numeric list + link within markdown syntax

**Comment by nesterone** [comment]
Ping, Ping, @bazzika , I'm waiting for updates in markdown

**Comment by nesterone** [comment]
so I can merge changes and you close the task

File: docs/spec.md [hunk 2]
```diff
@@ -33,10 +31,12 @@ In current iteration we are going to concentrate only on opening pre-build examp
 So user need a way to choose one spec from list of available examples it’s could be done by analogy with examples in Vega Editor (see Examples)
 To narrow down a scope we can disable examples which not supported yet
 
-/**
- * @todo #136:2d/UX Open Spec - list of screens in Zeplin
- *
- */
+
+1. (https://zpl.io/2yeQJ4q) - main view
```
Comment (nesterone) [inline docs/spec.md]: 1. fdfdf
1. fsdfsdfd
1. 24342344

**Pull request merged**